### PR TITLE
More tgui-next porting

### PR DESCRIPTION
**Pull request by ghost**
:cl: Multiple /tg/ coders
fix: tgui-next inputs now work on IE8 [actioninja]
add: The ore box now uses tgui-next [nightred]
refactor: Refactored tgui-next [stylemistake]
add: Made some plumbing machines use tgui-next inputs [actioninja]
add: Scanner gates now use tgui-next [nightred]
fix: ChemMaster no longer rounds volume of chemicals, allowing to package 0.1 unit pills. [stylemistake]
fix: Air Alarm layout will no longer jump around when microscopic amounts of gases are present (less than 0.01%). [stylemistake]
add: Gravity gen now uses tgui-next [Arkatos1]
add: Meteor shield controller now use tgui-next [kriskog]
add: Smoke machine now uses tgui-next [TheVekter]
add: The PANDEMIC now uses tgui-next [actioninja]
add: The portable air pump now used tgui-next [actioninja]
/:cl:

### CODE REVIEW

**Comment by Neotwr** [comment]
you should include it https://github.com/tgstation/tgstation/pull/48090

**Comment by ghost** [comment]
@Neo-0 
firefox_x9iSgADsj8

**Comment by Neotwr** [comment]
uh oh im dumb

File: code/datums/diseases/advance/symptoms/genetics.dm [hunk 1]
```diff
@@ -30,9 +30,12 @@ Bonus
 	symptom_delay_min = 60
 	symptom_delay_max = 120
 	var/no_reset = FALSE
-	threshold_desc = "<b>Resistance 8:</b> Causes two harmful mutations at once.<br>\
-					  <b>Stage Speed 10:</b> Increases mutation frequency.<br>\
-					  <b>Stealth 5:</b> The mutations persist even if the virus is cured."
+	threshold_descs = list(
+		"Resistance 8" = "The negative and mildly negative mutations caused by the virus are mutadone-proof (but will still be undone when the virus is cured if the resistance 14 threshold is not met).",
+		"Resistance 14" = "The host's genetic alterations are not undone when the virus is cured.",
```
Comment (ghost) [inline code/datums/diseases/advance/symptoms/genetics.dm:35]: Is this supposed to be here?

File: code/datums/diseases/advance/symptoms/narcolepsy.dm [hunk 2]
```diff
@@ -26,8 +26,11 @@ Bonus
 	var/sleep_level = 0
 	var/sleepy_ticks = 0
 	var/stamina = FALSE
-	threshold_desc = "<b>Transmission 7:</b> Also relaxes the muscles, weakening and slowing the host.<br>\
-					  <b>Resistance 10:</b> Causes narcolepsy more often, increasing the chance of the host falling asleep."
+	var/yawning = FALSE
```
Comment (ghost) [inline code/datums/diseases/advance/symptoms/narcolepsy.dm]: yawning is added, description is changed

File: code/datums/diseases/advance/symptoms/sneeze.dm [hunk 3]
```diff
@@ -27,8 +27,13 @@ Bonus
...
 	symptom_delay_min = 5
 	symptom_delay_max = 35
-	threshold_desc = "<b>Transmission 9:</b> Increases sneezing range, spreading the virus over a larger area.<br>\
-					  <b>Stealth 4:</b> The symptom remains hidden until active."
+	var/spread_range = 4
+	var/cartoon_sneezing = FALSE //ah, ah, AH, AH-CHOO!!
+	threshold_descs = list(
+		"Transmission 9" = "Increases sneezing range, spreading the virus over 6 meter cone instead of over a 4 meter cone.",
+		"Stealth 4" = "The symptom remains hidden until active.",
+		"Stage Speed 17" = "The force of each sneeze catapults the host backwards, potentially stunning and lightly damaging them if they hit a wall or another person mid-flight."
```
Comment (ghost) [inline code/datums/diseases/advance/symptoms/sneeze.dm]: not in the original description

File: tgui-next/packages/tgui/hotkeys.js [hunk 4]
```diff
@@ -172,9 +172,9 @@ const handleHotKey = (e, eventType, dispatch) => {
       // stack in order for this to be a fatal error.
       setTimeout(() => {
         throw new Error(
-          "OOPSIE WOOPSIE!! UwU We made a fucky wucky!! A wittle"
-          + " fucko boingo! The code monkeys at our headquarters are"
-          + " working VEWY HAWD to fix this!");
+          'OOPSIE WOOPSIE!! UwU We made a fucky wucky!! A wittle'
```
Comment (ghost) [inline tgui-next/packages/tgui/hotkeys.js:175]: fuck me bloody

**Review by ghost — CHANGES_REQUESTED** [review]
I didn't do much reviewing on the JS side considering we shouldn't have anything there which conflicts in any meaningful manner but it'd be good to sort out the byond inconsistencies

**Review by ghost — APPROVED** [review]

**Pull request merged**